modules/oadp-using-enable-api-group-versions.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: PROCEDURE
+[id="oadp-using-enable-api-group-versions_{context}"]
+= Working with different Kubernetes API versions in the same OADP operation
+
+By default, Velero only backs up resources that use the `preferredAPI` Kubernetes version. However, Velero also includes a feature, link:https://velero.io/docs/v1.9/enable-api-group-versions-feature/[Enable API Group Versions], that overcomes this limitation. When enabled on the source cluster, this feature causes Velero to back up _all_ supported Kubernetes API group versions on the cluster, not only the preferred one.
+
+[NOTE]
+====
+As of August 2022, Enable API is still in beta.
+====
+
+For example, consider the resource `horizontalpodautoscalers`, which falls under the `autoscaling` group and has 3 `apiVersions` - `v1`,`v2beta1`, and `v2beta2` - registered in the cluster, with the preferred API being `v1`.
+
+Without the Enable API Group Versions enabled, only the preferred API group version for `autoscaling`, `v1`, is backed up. With the feature enabled, the remaining supported versions, `v2beta1` and `v2beta2` are also backed up. Once the versions are stored in the backup tarball file, they will be available to be restored on the destination cluster.
```

Comment:
@RichardHoch 
"Without the Enable API Group Versions feature enabled, Velero backs up only the preferred API group version for the `autoscaling` group, `v1`. With the feature enabled, Velero backs up the remaining supported versions, `v2beta1` and `v2beta2`."
"...in the backup tarball file, they are available to be restored..."